schemas/regional/ch/AccountingSettings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "AccountingSettings",
+  "fields": [
+    {
+        "fieldname": "taxId",
+        "label": "Tax ID",
+        "fieldtype": "Data",
+        "placeholder": "CHE-123.456.789",
+        "section": "Default"
+    }    
+  ],
+  "quickEditFields": [
+    "fullname",
+    "email",
+    "companyName",
+    "country",
+    "currency",
+    "fiscalYearStart",
+    "fiscalYearEnd",
+    "taxId"
```

Comment:
I believe this list should be compared to

https://github.com/frappe/books/blob/8bf0a5f7362fbdcecf7d036061a3387adf889bb5/schemas/app/AccountingSettings.json#L125-L132

I suppose the field `currency` was copied from the Indian regional schema. Does it makes sense for Swizerland?

Reply:
While CHF is the national currency in Switzerland it is not uncommon for accounting to be done in EUR, especially if this is a company selling services in EU.

Reply:
If this is intentional, no problem, I supposed something like this. But if this was just copy and paste from Indian regional schema, perhaps this did not need to be included.